src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -99,6 +112,13 @@ case class Cluster(clusterName: ClusterName,
                    googleProject: GoogleProject,
                    googleServiceAccount: GoogleServiceAccount,
                    googleBucket: GcsBucketName,
+                   numberOfWorkers: Int,
+                   masterMachineType: String,
+                   masterDiskSize: Int,  //min 10
+                   workerMachineType: Option[String],
+                   workerDiskSize: Option[Int],   //min 10
+                   numberOfWorkerLocalSsds: Option[Int], //min 0 max 8
+                   numberOfPreemptibleWorkers: Option[Int] = None,
```

Comment:
Do we want a separate`WorkerConfig` object in ClusterRequest/Cluster to keep track of `workerMachineType`, `workerDiskSize`, `numberOfWorkerLocalSsds`, `numberOfPreemptibleWorkers`?